registry/auth/htpasswd/access.go

```diff
@@ -60,7 +53,35 @@ func (ac *accessController) Authorized(ctx context.Context, accessRecords ...aut
 		}
 	}
 
-	if err := ac.AuthenticateUser(username, password); err != nil {
+	// Dynamically parsing the latest account list
+	fstat, err := os.Stat(ac.path)
+	if err != nil {
+		return nil, err
+	}
+
+	lastModified := fstat.ModTime().String()
```

Comment:
Instead of converting to a string, you could do:

```go
lastModified := fstat.ModTime()
```

and

```go
if ac.modtime.Equal(lastModified) {
```